lib/jekyll-import/importers/mt.rb

```diff
@@ -58,15 +65,24 @@ def self.specify_options(c)
       def self.process(options)
         options  = default_options.merge(options)
 
+        engine   = options.fetch('engine')
         dbname   = options.fetch('dbname')
         user     = options.fetch('user')
         pass     = options.fetch('password', "")
         host     = options.fetch('host', "localhost")
+        port     = options.fetch('port')
```

Comment:
I think the issue you are highlighting here is that it might not work to pass  `:port => port` if `port` is not defined. I've updated the code so that now we only pass the `:port` key as a DB option if it is defined.    This seemed preferable to hardcoding the default ports for both MySQL and PostgreSQL into the code.